base/config_persistence.go

```diff
@@ -215,24 +205,23 @@ func (xbp *XattrBootstrapPersistence) loadConfig(ctx context.Context, c *gocb.Co
 	}
 }
 
-// Restore a deleted document's body.  Rewrites metadata, but preserves previous cfgCas
-func (xbp *XattrBootstrapPersistence) restoreDocumentBody(c *gocb.Collection, key string, value interface{}, cfgCas string) error {
+// Restore a deleted document's body.  Rewrites metadata
+func (xbp *XattrBootstrapPersistence) restoreDocumentBody(c *gocb.Collection, key string, value interface{}, cas gocb.Cas) (casOut gocb.Cas, err error) {
```

Comment:
We aren't using a cas check here because there aren't cas checks on insert. I think you can remove `cas` from an argument here.

 We are sure that there is no document existing because on `gocb.StoreSemanticsInsert`